src/test/parse-fail/mod_file_not_exist.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-windows
```

Comment:
I now see that there is another test in this PR, `mod_file_not_exist_windows.rs`, which essentially does this. But it's done by exhausting every non-Windows platform. Is that the right way to do this?